shared/test/test_net_sim_api.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/autorun'
+require 'rack/test'
+require File.expand_path '../../../deployment', __FILE__
+require File.expand_path '../../middleware/net_sim_api', __FILE__
+
+ENV['RACK_ENV'] = 'test'
```

Comment:
If I remember correctly, our `channels_api_secret` ([used here](https://github.com/code-dot-org/code-dot-org/blob/staging/shared/middleware/helpers/storage_id.rb#L18)) is one value for development and a second value for staging/test/production - hence only two public keys here.  This configuration is managed through Chef, though I must admit that whole system is still somewhat opaque to me.